app/Http/Controllers/UsersController.php

```diff
@@ -12,8 +13,22 @@ class UsersController extends Controller
      */
     protected $northstar;
 
-    public function __construct(Northstar $northstar)
+    public function __construct(Registrar $registrar)
     {
-        $this->northstar = $northstar;
+        $this->registrar = $registrar;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $ids = User::all()->pluck('northstar_id')->all();
```

Comment:
oh my bad, I thought this was like gladiator, where we had to kill the whole user page